tracegen/internal/tracegen/config.go

```diff
@@ -36,8 +36,9 @@ type Config struct {
 	ServiceName      string
 
 	// OTLP config
-	Endpoint string
-	Insecure bool
+	Endpoint     string
+	Insecure     bool
+	HttpExp bool
```

Comment:
```suggestion
	HTTPExp bool
```

Or perhaps `UseHTTP`?